addons/web/static/tests/fields/relational_fields/field_one2many_tests.js

```diff
@@ -123,7 +123,6 @@ QUnit.module('fields', {}, function () {
                         display_name: "leonardo",
                         turtle_bar: true,
                         turtle_foo: "yop",
-                        turle_int: 1,
```

Comment:
this one is ok :D we also spotted it in an ongoing task, that might massively conflict with this one btw ^^